### PR TITLE
AWS deployment

### PR DESCRIPTION
**Pull request by neimanchen**

### CODE REVIEW

File: src/client/components/AddItem/UploadItemInfo.jsx [hunk 1]
```diff
@@ -21,7 +21,9 @@ export class UploadItemInfo extends React.Component {
   }
 
   closeModal() {
+    console.log('CLOSED')
```
Comment (skoshkarli) [inline src/client/components/AddItem/UploadItemInfo.jsx]: console log

File: src/client/components/AddItem/UploadItemInfo.jsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ export class UploadItemInfo extends React.Component {
 
   closeModal() {
     this.props.actions.updateModalState(false);
+
```
Comment (skoshkarli) [inline src/client/components/AddItem/UploadItemInfo.jsx]: empty line

File: src/client/components/AddItem/UploadItemInfo.jsx [hunk 3]
```diff
@@ -63,18 +64,16 @@ export class UploadItemInfo extends React.Component {
...
-          open={this.props.modalOpen}
+          open={this.getState()}
           style={inlineStyle.modal}
         >
           <Modal.Content image>
             <Image wrapped size='large' src={this.props.imageURL} />
             <Modal.Description>
               <UploadForm onSubmit={this.submitForm} styles={this.props.styles} colors={this.props.colors} close={this.closeModal}/>
             </Modal.Description>
-            <Button onClick={this.closeModal}>Close</Button>
```
Comment (skoshkarli) [inline src/client/components/AddItem/UploadItemInfo.jsx]: you dont need to update your state to close on click?

Comment (neimanchen) [reply src/client/components/AddItem/UploadItemInfo.jsx]: that was a test button, i have it passed to upload form on line 75 already

**Pull request merged**